desktop/cmp/button/Button.scss

```diff
@@ -370,3 +370,22 @@
     }
   }
 }
+
+//------------------------
+// Header "tool" buttons in panels and docks
+// Adjustment needed only in light theme.
+//------------------------
+*:not(.xh-dark),
+*:not(.bp3-dark) {
+  .xh-dock-view__header,
+  .xh-panel-header__items {
+    .xh-button.bp3-button.xh-button--minimal {
+      &.xh-button--intent-none {
```

Comment:
I followed the suggestion from PanelHeader.scss, where we don't worry about other intents or styles other than minimal when header buttons are concerned.